products/content/tier-2-content-IA-and-design/cope-cms-faq-library/engineering/README.md

```diff
@@ -1,5 +1,33 @@
 # Exposing Q&A on New Content Types
 
+## CMS Instructions: Reusable Q&A Implementation Runbook
+For existing content types, edit the field_content_block field (Main content) if it exists on the content type, select to add the Q&A paragraph type, then save.
+
+Now when editors want to add an existing reusable Q&A they will see it as one of the options for adding main content.
+
+1. In the Drupal UI > Structure > Content types and select your content type.
+2. Edit the chosen Entity Reference Revisions field or add a new field, type Paragraph under Reference Revisions.
+3. If editing existing field, skip the following step. If it's a new field, select Paragraph for "Type of item to reference" and select Unlimited cardinality/allowed number of values.
```

Comment:
CONSIDER:
The if/then scenarios could cause some confusion. Consider breaking the steps into distinct groups: creating a new field, and utilizing an existing field. There might be some redundancies, but I think it is worth that for extra clarity/ease of following.